debian/base/Dockerfile

```diff
@@ -37,11 +40,12 @@ ENV SCALA_VERSION="2.11"
 ENV CONFLUENT_MAJOR_VERSION="3"
 ENV CONFLUENT_MINOR_VERSION="3"
 
-ENV CONFLUENT_PATCH_VERSION="0"
+ENV CONFLUENT_PATCH_VERSION="0~SNAPSHOT"
 ENV CONFLUENT_VERSION="$CONFLUENT_MAJOR_VERSION.$CONFLUENT_MINOR_VERSION.$CONFLUENT_PATCH_VERSION"
 ENV CONFLUENT_DEB_VERSION="1"
 
-ENV KAFKA_VERSION="0.11.0.0-SNAPSHOT"
+# Add ~SNAPSHOT (with tilde) for internal releases
+ENV KAFKA_VERSION="0.11.0.0~SNAPSHOT"
```

Comment:
I'd suggest making all the params that vary across releases build args, not setting any defaults for them in the Dockerfiles, and use the Makefile as you only place you need to change them.